<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:qlineargradient(spread:pad, x1:0.254, y1:0.267, x2:1, y2:1, stop:0.580488 rgba(30, 100, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>9</y>
     <width>1771</width>
     <height>881</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>메인화면</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>551</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 36pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>경찰서 조회</string>
     </property>
    </widget>
    <widget class="QPushButton" name="police_pushButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>110</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 18pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="police_tableWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>901</width>
       <height>301</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 10pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>지방청</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>경찰서</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>관서명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>전화번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주소</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="police_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>731</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 75 25pt &quot;Sitka Banner&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="modify_pushButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>50</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 18pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>수정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_delete_pushButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>520</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 18pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>추가 및 삭제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="police_graph_pushButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>50</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 18pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>그래프</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>480</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 36pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>경찰서별 범죄 조회</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="crime_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>580</y>
       <width>731</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 75 25pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="crime_pushButton">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>580</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 18pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QPushButton" name="crime_graph_pushButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>520</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 18pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>그래프</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="crime_tableWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>650</y>
       <width>901</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 10pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>발생연도</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>경찰서</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>살인</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>강도</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>절도</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>폭력</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>수정</string>
    </attribute>
    <widget class="QLabel" name="label_55">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>지방청 목록</string>
     </property>
    </widget>
    <widget class="QComboBox" name="district_office_combobox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_61">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>경찰서 목록</string>
     </property>
    </widget>
    <widget class="QComboBox" name="police_combobox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_62">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>관서명(세부지역)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="detail_combobox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_63">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>경찰_구분</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="police_division_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>450</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_64">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>경찰_전화번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="police_call_number_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>570</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_65">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>640</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>경찰_주소</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="police_address_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>700</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="modify_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>770</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>데이터 수정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gomain_button">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>770</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>메인화면</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="modify_tableWidget">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>621</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 10pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>지방청</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>경찰서</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>관서명</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>구분</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>전화번호</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>주소</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>추가,삭제</string>
    </attribute>
    <widget class="QLabel" name="label_52">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>범죄 유형</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="crime_category_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_53">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>범죄 건수</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="crime_number_lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="add_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>450</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>추가</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_54">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);
font: 75 italic 30pt &quot;Sitka Banner&quot;;
</string>
     </property>
     <property name="text">
      <string>경찰서 목록</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="chage_tableWidget">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>1341</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 10pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>발생연도</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>경찰서</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>살인</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>강도</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>절도</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>폭력</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="delete_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>450</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>삭제</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gotomain_button">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>450</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius: 20px;
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
color: rgb(0, 0, 0)
}</string>
     </property>
     <property name="text">
      <string>메인화면</string>
     </property>
    </widget>
    <widget class="QLabel" name="add_delete_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>341</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:rgb(255, 255, 255);
font: 20pt &quot;Cambria&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>그래프</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
